packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -264,7 +239,7 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                         Clearing all items is still possible since this component is controlled. It just not cannot
                         be through the default button in this component, rather done through a custom button that can be
                         rendered from within the popover through the itemListRenderer or from externally. */}
-                        {this.props.customTarget != null && popoverInputForCustomTarget}
+                        {this.props.customTarget != null && this.getTagInput(listProps, CoreClasses.FILL)}
```

Comment:
When the TagInput is rendered inside the Popover, grow the TagInput to be the width of the Popover by default